.github/workflows/build-latex.yml

```diff
@@ -1,5 +1,5 @@
 name: Build LaTeX document
-on: [push]
+on: [push, pull_request]
```

Comment:
@Sammers21 I'd create two different workflows for build and release, build just test that PDF can be built, and release for PDF uploading